test/Scripts.Integration.Test/Editor/BuildTimeOptions.cs

```diff
@@ -11,33 +12,10 @@ public override void Configure(SentryUnityOptions options, SentryCliOptions cliO
     {
         Debug.Log("Sentry: BuildTimeOptions::Configure() called");
 
-        switch (EditorUserBuildSettings.selectedBuildTargetGroup)
-        {
-            case BuildTargetGroup.Android:
-                options.Dsn = "http://publickey@10.0.2.2:8000/12345";
-                break;
-            case BuildTargetGroup.WebGL:
-                options.Dsn = "http://publickey@127.0.0.1:8000/12345";
-                break;
-            default:
-                options.Dsn = "http://publickey@localhost:8000/12345";
-                break;
-        }
-
-        Debug.LogFormat("Sentry: Setting options.Dsn = {0}", options.Dsn);
-
-        options.Il2CppLineNumberSupportEnabled = true;
-        options.Debug = true;
-        options.DiagnosticLevel = SentryLevel.Debug;
-        options.TracesSampleRate = 1.0d;
-        options.PerformanceAutoInstrumentationEnabled = true;
-
-        cliOptions.UploadSymbols = !string.IsNullOrEmpty(cliOptions.UrlOverride);
-        cliOptions.UploadSources = cliOptions.UploadSymbols;
-        cliOptions.Organization = "sentry-sdks";
-        cliOptions.Project = "sentry-unity";
-        cliOptions.Auth = "dummy-token";
-
+        // Make sure the config is still called and the DSN has been set
+        Assert.IsFalse(string.IsNullOrWhiteSpace(options.Dsn));
```

Comment:
seems like a downgrade where we don't assert the actual value anymore, just that it's not set.

If we somehow end up passing: "Disable SDK" which is `""`, tests would pass.

Also, if test fails we'd get: "Expected true but got false". So ideally use a better assertion API like "Assert.Same(A,B)" or whatever this lib has (NUnit is it?)

Reply:
Those are the old config callbacks for the smoke test. All I want to check there is that those are getting called.
1. Invoke the "new way" to programmatic Configure and set the options for the smoke tests
2. Make sure the "old way" is still getting called after the new one. So we stay compatible and won't break users.